Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -l
+# Download and install https://github.com/naumanjaved/fingerprint_maps
+# Then run this bash script for all chromosomes except for Y
+CHR="22"
```

Comment:
this bash script wont work as is.

maybe can add a for loop across all chromosome, 
also maybe add mkdirs for missing directoriess

Reply:
do I need GATK installed? do I need a specific version?

Reply:
also I need bcftools, wget, python (any version?)
also this file is not here I think: ~/Data/Fingerprinting/fingerprint_maps/build_fingerprint_maps.py

Reply:
ok let's put it in a folder: Fingerprintin../tasks/ to make differentiate it from non One-Off tasks.